### PR TITLE
Remove old comment

### PR DESCRIPTION
**Pull request by crice100**
The referenced ticket is closed, and we're now on python3. Also fix some typos.

https://openedx.atlassian.net/browse/PLAT-2287

### CODE REVIEW

File: tubular/scripts/retirement_partner_report.py [hunk 1]
```diff
@@ -4,11 +4,6 @@
 """
 Command-line script to drive the partner reporting part of the retirement process
 """
-# NOTE: Make sure that all non-ascii text written to standard output (including
-# print statements and logging) is manually encoded to bytes using a utf-8 or
-# other encoding.  We currently make use of this script within a context that
-# does NOT tolerate unicode text on sys.stdout, namely python 2 on Build
-# Jenkins.  PLAT-2287 tracks this Tech Debt.
```
Comment (doctoryes) [inline tubular/scripts/retirement_partner_report.py:7]: Has build.testeng.edx.org and its jobs been converted to use Python 3 now, @estute ?

**Review by doctoryes — COMMENTED** [review]
One question to answer, otherwise 👍

**Review by staubina — APPROVED** [review]

File: tubular/scripts/retirement_partner_report.py (hunk 1 above)
Comment (estute) [reply tubular/scripts/retirement_partner_report.py:7]: All of the jobs within the user-retirement process use python3.5 virtualenvs on build.testeng.edx.org. This is the part pertinent to this change:
* https://github.com/edx/jenkins-job-dsl/blob/master/platform/jobs/RetirementJobs.groovy#L474-L477
* https://github.com/edx/jenkins-job-dsl/blob/master/platform/jobs/RetirementJobs.groovy#L602-L605

Comment (doctoryes) [reply tubular/scripts/retirement_partner_report.py:7]: Excellent! Thanks, @estute .

**Review by doctoryes — APPROVED** [review]

**Pull request merged**